RecoJets/JetAlgorithms/src/CMSBoostedTauSeedingAlgorithm.cc

```diff
@@ -84,7 +87,7 @@ namespace contrib{
   }
 
   std::pair<PseudoJet, PseudoJet> CMSBoostedTauSeedingAlgorithm::findSubjets(const PseudoJet& jet, int depth, bool& subjetsFound) const 
-  {
+  { float etaMax_=3; //CG: cut on eta subjets <3. Anyways taus are cut at 2.3 
```

Comment:
Please put this definition on its own line separate from the brace. Also, use `const`:
`const float etaMax_ = 3.0;`